rpmautospec/subcommands/process_distgit.py

```diff
@@ -101,7 +101,11 @@ def process_distgit(
 
     autorelease_number = result["release-number"]
 
-    with processor.specfile.open("r") as specfile, tempfile.NamedTemporaryFile("w") as tmp_specfile:
+    with processor.specfile.open(
+        "r", encoding="utf-8", errors="surrogateescape"
+    ) as specfile, tempfile.NamedTemporaryFile(
+        "w", encoding="utf-8", errors="surrogateescape"
+    ) as tmp_specfile:
```

Comment:
Minor nitpick: is there any way that you can split this up more readably? Currently, it takes effort to recognize that there are two file objects being created here, because of the placement of the comma on line 106.

Reply:
Hmm, [parenthesized context managers](https://docs.python.org/3/whatsnew/3.10.html#summary-release-highlights) need Python 3.10… 🤔

Reply:
Okay, I guess not. Pity.

Reply:
Hmm. Technically, [Python 3.10 is not needed](https://github.com/python/peps/pull/2244#discussion_r785512543), so the question is more how much do we care about Python 3.9 on non-CPython…

Reply:
It's very much not worth worrying about it; as I said, it was a nitpick. I've already approved the rest of the patch :-)